test/pkg/steps/deploy_files.go

```diff
@@ -18,33 +18,48 @@ import (
 	"fmt"
 
 	api "github.com/kiegroup/kogito-operator/apis"
+	corev1 "k8s.io/api/core/v1"
 
 	"github.com/cucumber/godog"
 	"github.com/kiegroup/kogito-operator/test/pkg/config"
 	"github.com/kiegroup/kogito-operator/test/pkg/framework"
+
+	operatorframework "github.com/kiegroup/kogito-operator/core/framework"
 )
 
 const sourceLocation = "src/main/resources"
 
 func registerKogitoDeployFilesSteps(ctx *godog.ScenarioContext, data *Data) {
 	// Deploy steps
 	ctx.Step(`^Deploy (quarkus|springboot) file "([^"]*)" from example service "([^"]*)"$`, data.deployFileFromExampleService)
+	ctx.Step(`^Deploy (quarkus|springboot) file "([^"]*)" from example service "([^"]*)" with env:$`, data.deployFileFromExampleServiceWithEnv)
 	ctx.Step(`^Deploy (quarkus|springboot) folder from example service "([^"]*)"$`, data.deployFolderFromExampleService)
 }
 
 // Deploy steps
 
 func (data *Data) deployFileFromExampleService(runtimeType, file, serviceName string) error {
 	sourceFilePath := fmt.Sprintf(`%s/%s/%s/%s`, data.KogitoExamplesLocation, serviceName, sourceLocation, file)
-	return deploySourceFilesFromPath(data.Namespace, runtimeType, serviceName, sourceFilePath)
+	return deploySourceFilesFromPathWithEnv(data.Namespace, runtimeType, serviceName, sourceFilePath, nil)
```

Comment:
You could also call the new function (`deployFileFromExampleServiceWithEnv`) from here with empty table and reusing the formatting code. Up to you.

Reply:
I would say that it does not change much for now. Let's see in case we would need to reuse the env part somehow